### ISSUE TITLE
Running script causes error

### ISSUE THREAD

**Issue by BubblegumZen** [circleguard/circleguard#183]
I was following the `Building from source` segment as it is what was recommended for Linux but I got this error.
I used wine to run the windows executable for x86 and that worked but I would still like to run the python script as it's more convenient.

- **Checked Previous closed issues to ensure this is not a duplicate.**
- **Python Version**: `Python 3.10.0`
- **Output of `pip show`**:

```
Package            Version
------------------ ---------
certifi            2021.10.8
charset-normalizer 2.0.12
circleguard        5.2.6
circlevis          1.4.6
cycler             0.11.0
idna               3.3
kiwisolver         1.4.0
matplotlib         3.2.2
numpy              1.22.3
oauthlib           3.2.0
osrparse           6.0.0
ossapi             2.2.3
packaging          21.3
pip                22.0.4
pkg_resources      0.0.0
portalocker        2.2.1
pyparsing          3.0.7
PyQt5              5.15.2
PyQt5-sip          12.9.1
python-dateutil    2.8.2
requests           2.27.1
requests-oauthlib  1.3.1
scipy              1.8.0
setuptools         60.10.0
six                1.16.0
slider             0.5.4
typing-utils       0.1.0
urllib3            1.26.9
wheel              0.37.1
wtc                1.2.1
```
**The error**:

```py
(myvenv) ┌─[bubble@pop-os] - [~/Desktop/circleguard] - [433]
└─[$] python circleguard/main.py                                              [18:35:07]
Traceback (most recent call last):
  File "/home/bubble/Desktop/circleguard/circleguard/main.py", line 211, in <module>
    welcome = CircleguardWizard()
  File "/home/bubble/Desktop/circleguard/circleguard/wizard.py", line 26, in __init__
    self.addPage(IntroPage())
  File "/home/bubble/Desktop/circleguard/circleguard/wizard.py", line 85, in __init__
    super().__init__()
  File "/home/bubble/Desktop/circleguard/circleguard/wizard.py", line 17, in __init__
    image = QPixmap(resource_path("logo/logo.png")).scaled(QSize(banner.height() * 0.85, banner.height() * 0.85), transformMode=Qt.SmoothTransformation)
TypeError: arguments did not match any overloaded call:
  QSize(): too many arguments
  QSize(int, int): argument 1 has unexpected type 'float'
  QSize(QSize): argument 1 has unexpected type 'float'
Traceback (most recent call last):
  File "/usr/lib/python3.10/weakref.py", line 667, in _exitfunc
    f()
  File "/usr/lib/python3.10/weakref.py", line 591, in __call__
    return info.func(*info.args, **(info.kwargs or {}))
  File "/home/bubble/Desktop/circleguard/myvenv/lib/python3.10/site-packages/circleguard/circleguard.py", line 744, in _cleanup
    library.close()
  File "/home/bubble/Desktop/circleguard/myvenv/lib/python3.10/site-packages/slider/library.py", line 95, in close
    self._db.close()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 139786874508864 and this is thread id 139787629551552.
[1]    33217 segmentation fault (core dumped)  python circleguard/main.py
```

**Comment by tybug** [comment]
We do indeed seem to be violating the signature of `QSize` - it expects integer values, and we're providing it floats. macos and windows doesn't seem to care about this violation, but linux apparently does, for whatever reason.

I've pushed a fix (e5b3635bd1196dfd8c052e058470b836e958b5d4) - can you double check this fixes the issue for you?

**Comment by tybug** [comment]
I can actually reproduce on python 3.10, so this isn't an OS issue. Even though our `PyQt5` versions match, the bindings must have changed between python versions. The commit above does fix this issue for me, so I'm going to close as resolved. 

Thanks for the report, and let us know if you run into any more issues while using python3.10 - we build with python 3.9 and I'm not aware of anyone else running circleguard locally with python3.10, so we don't have much coverage on 3.10.
